src/components/BottomWidgetsContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import React       from 'react';
+import PropTypes   from 'prop-types';
+import { connect } from '../store/Connect';
+
+import '../../sass/components/_bottom-widget-container.scss';
+
+
+class BottomWidgetsContainer extends React.Component {
+    componentDidUpdate() {
+        if (React.Children.count(this.props.children)) {
+            this.props.updateChartMargin(200);
```

Comment:
@morteza-binary doesn't seems right to use a fix number here. i used it in last digit cause i knew the height of last digit component but what if user add another component?

Reply:
@MahboobehMohammadi IMO, it's better to keep the height of the bottom widget fix. 
@ashkanx What's your idea? 

BTW, If we want to calculate the wide of the children and use it here, we have to use `ref` for each element in the children to get its height that IMO, it is not a good idea.

Reply:
I think it can't be a fixed value eventually. it should be relative to screen size and other active components in the screen. but probably we should cover that in another card including UI team as well.

Reply:
@morteza-binary why just not use a Height prop so anybody use it just set the height. It’s not like it contains dynamic children.